eng/restore/binplacePackages/binplacePackages.depproj

```diff
@@ -26,21 +26,17 @@
     </BinPlaceConfiguration>
   </ItemGroup>
 
-  <ItemGroup>
+  <!-- This is to make sure that nothing unintended gets restored for netcoreapp. -->
+  <ItemGroup Condition="'$(TargetGroup)' != 'netcoreapp'">
     <!-- runtime dependency: System.Data.SqlClient uap10.0.16299 -->
     <!-- runtime dependency: System.Diagnostics.PerformanceCounters netcoreapp2.0,net461 -->
-    <PackageReference Include="System.Memory" Condition="'$(TargetFramework)' == 'netcoreapp2.0' OR '$(TargetFramework)' == 'uap10.0.16299' OR '$(TargetsNetfx)' == 'true' OR ('$(TargetsNetStandard)' == 'true' AND '$(NETStandardVersion)' &gt;= 1.1)">
-      <Version>4.5.3</Version>
-    </PackageReference>
-
-    <PackageReference Include="System.Numerics.Vectors" Condition="'$(TargetsNetStandardLowerThan21)' == 'true' OR '$(TargetsNetfx)' == 'true'">
-      <Version>4.5.0</Version>
-    </PackageReference>
-
-    <PackageReference Include="System.Threading.Tasks.Extensions" Condition="'$(TargetFramework)' == 'netcoreapp2.0' OR '$(TargetsNetfx)' == 'true' OR '$(TargetsNetStandard)' == 'true'">
-      <Version>4.5.2</Version>
-    </PackageReference>
-
+    <PackageReference Include="System.Memory" Version="4.5.2" Condition="'$(TargetFramework)' == 'netcoreapp2.0' OR '$(TargetFramework)' == 'uap10.0.16299' OR '$(TargetsNetfx)' == 'true' OR ('$(TargetsNetStandard)' == 'true' AND '$(NETStandardVersion)' &gt;= 1.1)" />
```

Comment:
Looks like this regressed System.Memory ref, discovered here: https://github.com/dotnet/sdk/pull/3884